features/step_definitions/networking.rb

```diff
@@ -716,7 +716,7 @@
   end
   # OVN uses `br-ex` and `-` is not a word char, so we have to split on whitespace
   cb[cb_name] = @result[:response].split("\n").first.split[4]
-  logger.info "The node's default interface is stored in the #{cb_name} clipboard."
+  logger.info "The node's default interface is stored in the #{cb_name} clipboard as #{cb[cb_name]}."
```

Comment:
adding `as <variable>` to the logger. This helps readability of the logs to determine if the correct value is being saved to the clipboard. Very much improvement in Quality of Life with this addition.